apps/meteor/tests/e2e/omnichannel/omnichannel-livechat-queue-management.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { createFakeVisitor } from '../../mocks/data';
+import { IS_EE } from '../config/constants';
+import { createAuxContext } from '../fixtures/createAuxContext';
+import { Users } from '../fixtures/userStates';
+import { HomeOmnichannel, OmnichannelLiveChat } from '../page-objects';
+import { test, expect } from '../utils/test';
+
+const firstVisitor = createFakeVisitor();
+
+const secondVisitor = createFakeVisitor();
+
+test.use({ storageState: Users.user1.state });
+
+test.describe('OC - Livechat - Queue Management', () => {
+	test.skip(!IS_EE, 'Enterprise Only');
+
+	let poHomeOmnichannel: HomeOmnichannel;
+	let poLiveChat: OmnichannelLiveChat;
+
+	const waitingQueueMessage = 'This is a message from Waiting Queue';
+	const queuePosition1 = 'Your spot is #1';
+	const queuePosition2 = 'Your spot is #2';
+
+	test.beforeAll(async ({ api, browser }) => {
+		const responses = await Promise.all([
+			api.post('/settings/Livechat_Routing_Method', { value: 'Manual_Selection' }),
+			api.post('/settings/Livechat_waiting_queue', { value: true }),
+			api.post('/settings/Livechat_waiting_queue_message', { value: waitingQueueMessage }),
+			api.post('/livechat/users/agent', { username: 'user1' }),
+		]);
+
+		responses.forEach((res) => expect(res.status()).toBe(200));
+
+		const { page: omniPage } = await createAuxContext(browser, Users.user1, '/', true);
+		poHomeOmnichannel = new HomeOmnichannel(omniPage);
+	});
+
+	test.beforeEach(async ({ browser, api }) => {
+		const context = await browser.newContext();
+		const page2 = await context.newPage();
+
+		poLiveChat = new OmnichannelLiveChat(page2, api);
+		await poLiveChat.page.goto('/livechat');
+	});
+
+	test.afterAll(async ({ api }) => {
+		const responses = await Promise.all([
+			api.post('/settings/Livechat_Routing_Method', { value: 'Auto_Selection' }),
+			api.post('/settings/Livechat_waiting_queue', { value: false }),
+			api.post('/settings/Livechat_waiting_queue_message', { value: '' }),
+			api.delete('/livechat/users/agent/user1'),
+		]);
+
+		responses.forEach((res) => expect(res.status()).toBe(200));
+	});
+
+	test.afterEach(async () => {
+		await poLiveChat.closeChat();
+	});
+
+	test('OC - Livechat - Queue Management - Change position on Queue', async ({ browser, api }) => {
+		let poLiveChat2: OmnichannelLiveChat;
+
+		await test.step('should start secondary livechat session', async () => {
+			const context = await browser.newContext();
+			const page = await context.newPage();
+			poLiveChat2 = new OmnichannelLiveChat(page, api);
```

Comment:
You need to close this page after the test, I would recommend moving the creation to the beforeEach callback, if you need 2 pages on only one test you can add an additional describe within this describe and have a beforeEacho for only this test